### PR TITLE
Update log output of testsuite management

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

**Review by HolQue — CHANGES_REQUESTED** [review]
Please consider also:
(1)
I would prefer to have all logged paths encapsulated in single quotes (like also done at other positions).

Why have you changed from `f"..."` to `f'...'`?

What I mean is:

Not: `Suite Path: C:\RobotTest\projects\ROBFW_AIO\RobotFramework_AIO\test\tutorial-test\playground\`

But: `Suite Path: 'C:\RobotTest\projects\ROBFW_AIO\RobotFramework_AIO\test\tutorial-test\playground\'`

**Comment by test-fullautomation** [comment]
:-)

Hi Son,
in general we put since TML all dynamic parts of logged data in single quotes.

E.g.
```print(f"My dynamic data: '{myData}'"```

This helps to separate the text from the displayed data.
It also helps to see e.g. wrong spaces, ... in the dynamic data.
So, please change your strings back to double quotes and set the dynamic parts in single quotes.

This is a general design rule for all output data for screen as well as log files.

Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
please read my findings.
Thank you,
 Thomas

File: RobotFramework_TestsuitesManagement/Keywords/CSetup.py [hunk 1]
```diff
@@ -102,12 +102,12 @@ def testsuite_setup(self, sTestsuiteCfgFile=''):
             logger.warn(f"Running with configuration level 4! \nSelected configuration file '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}'")
 
         RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.verifyRbfwVersion()
-        logger.info(f'Suite Path: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestcasePath}')
-        logger.info(f'CfgFile Path: {RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}')
+        logger.info(f"Suite Path: '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestcasePath}'")
```
Comment (test-fullautomation) [inline RobotFramework_TestsuitesManagement/Keywords/CSetup.py:105]: Perfect!

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks good to me!
Thank you,
 Thomas

**Pull request merged**